multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -23,6 +23,9 @@ coreos:
             [Unit]
             Requires=flanneld.service
             After=flanneld.service
+            [Service]
+            ExecStart=
+            ExecStart=/usr/lib/coreos/dockerd daemon --host=fd:// $DOCKER_OPTS $DOCKER_CGROUPS $DOCKER_OPT_MTU
```

Comment:
[Relevant section](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=) for Euan and Colin's points.

> Settings from these EnvironmentFiles override settings made with Environment=. If the same variable is set twice from EnvironmentFiles, the files will be read in the order they are specified and the later setting will override the earlier setting.